src/PersonObjects/Person.ts

```diff
@@ -1,17 +1,19 @@
 import type { Skills } from "./Skills";
 import type { HP } from "./HP";
-import type { Person as IPerson } from "@nsdefs";
+import type { Person as IPerson } from "@nsdefs"; // Importing Person interface from namespace @nsdefs
 
-import * as personMethods from "./PersonMethods";
-import { CityName } from "@enums";
-import { PlayerOwnedAugmentation } from "../Augmentation/PlayerOwnedAugmentation";
-import { calculateSkill } from "./formulas/skill";
-import { defaultMultipliers } from "./Multipliers";
-import { IReviverValue } from "../utils/JSONReviver";
+import * as personMethods from "./PersonMethods"; // Importing methods for managing person-related actions
+import { CityName } from "@enums"; // Importing CityName enum
+import { PlayerOwnedAugmentation } from "../Augmentation/PlayerOwnedAugmentation"; // Importing PlayerOwnedAugmentation class
+import { calculateSkill } from "./formulas/skill"; // Importing function for calculating skill levels
+import { defaultMultipliers } from "./Multipliers"; // Importing default multipliers
+import { IReviverValue } from "../utils/JSONReviver"; // Importing interface for JSON reviver values
 
 // Base class representing a person-like object
 export abstract class Person implements IPerson {
-  hp: HP = { current: 10, max: 10 };
+  hp: HP = { current: 10, max: 10 }; // Current and maximum hit points of the person.
```

Comment:
Sorry if this comes across as a little harsh, but this is IMO a classic example of "just restating the code."

I.e. the comment is not saying anything here that the code does not already say; the variable and field names already say that these are `current` and `max` HP (a very well-understood abbreviation for hit points, honestly the abbreviation is better known than the full phrase).

Most of the time, you want the code to be self-explanatory like this, so that comments aren't needed. Where comments *should* be added are where something tricky is going on, and the code itself is not enough. Even then, the comment should not *restate* what the code does, but rather explain the *motivation* for the code, which is usually the context that is missing.

(Applies to other files as well)